tests/sentry/api/endpoints/test_chunk_upload.py

```diff
@@ -131,7 +131,7 @@ def test_too_large_request(self):
         assert response.status_code == 200, response.content
 
         # We overflow the request here
-        files.append(SimpleUploadedFile(sha1("content").hexdigest(), "content"))
+        files.append(SimpleUploadedFile(sha1(b"content").hexdigest(), "content"))
```

Comment:
This line seems repeated quite a lot. Maybe it is wiser to abstract that out with `content.encode('utf-8')` and leave the strings as they are?

Reply:
Hmm, I'm somewhat indifferent.